timbuctoo-instancev4/src/test/java/nl/knaw/huygens/timbuctoo/experimental/exports/excel/sheet/EntitySheetTest.java

```diff
@@ -0,0 +1,174 @@
+
+    EntitySheet instance = new EntitySheet(collection, workbook, graphWrapper, mappings, relationTypes, acceptedProp);
+
+    instance.renderToWorkbook(graph.traversal().V().toSet());
+
+    SXSSFSheet sheet = workbook.getSheet("wwpersons");
+    assertThat(sheet, instanceOf(SXSSFSheet.class));
+    assertThat(sheet.getRow(0).getCell(0).getStringCellValue(), equalTo("tim_id"));
+    assertThat(sheet.getRow(1).getCell(0).getStringCellValue(), equalTo("uuid"));
+    assertThat(sheet.getRow(2).getCell(0), equalTo(null));
+
+    assertThat(sheet.getRow(0).getCell(1), equalTo(null));
+  }
+
+
+  @Test
+  public void renderToWorkbookRendersTheVerticesAsExcelCellsToTheSheet() throws IOException {
+    final GraphWrapper graphWrapper = mock(GraphWrapper.class);
+    final SXSSFWorkbook workbook = new SXSSFWorkbook();
+    final Graph graph = newGraph()
+      .withVertex("v1", v -> {
+        v.withProperty("tim_id", "123");
+        v.withProperty("types", jsnA(jsn("person"), jsn("wwperson")).toString());
+        v.withProperty("wwperson_tempName", "temp name");
+        v.withProperty("wwperson_names", jsnO("list", jsnA(
+          jsnO("components", jsnA(
+            jsnO("type", jsn("FORENAME"), "value", jsn("foreName 1")),
+            jsnO("type", jsn("SURNAME"), "value", jsn("surName 1"))
+          )),
+          jsnO("components", jsnA(
+            jsnO("type", jsn("FORENAME"), "value", jsn("foreName 2")),
+            jsnO("type", jsn("SURNAME"), "value", jsn("surName 2"))
+          ))
+        )).toString());
+      })
+      .withVertex("v2", v -> {
+        v.withProperty("tim_id", "234");
+        v.withProperty("types", jsnA(jsn("person"), jsn("wwperson")).toString());
+        v.withProperty("wwperson_tempName", "tmp 2");
+        v.withOutgoingRelation("isRelatedTo", "v1", r -> {
+          r.withIsLatest(true);
+          r.withAccepted("wwrelation", true);
+          return r;
+        });
+      })
+      .build();
+    given(graphWrapper.getGraph()).willReturn(graph);
+    EntitySheet instance = new EntitySheet(collection, workbook, graphWrapper, mappings, relationTypes, acceptedProp);
+
+    instance.renderToWorkbook(graph.traversal().V().toSet());
+    SXSSFSheet sheet = workbook.getSheet("wwpersons");
+
+    List<String> firstHeaderValues = new ArrayList<>();
+    Map<String, Integer> columnIndices = new HashMap<>();
+    for (int i = 0; i < 7; i++) {
+      SXSSFCell cell = sheet.getRow(0).getCell(i);
+      if (cell != null) {
+        firstHeaderValues.add(cell.getStringCellValue());
+        columnIndices.put(cell.getStringCellValue(), i);
+      } else {
+        firstHeaderValues.add("");
+      }
+    }
+    assertThat(firstHeaderValues, containsInAnyOrder(Lists.newArrayList(
+      equalTo(""), equalTo(""), equalTo(""),
+      equalTo("tim_id"), equalTo("tempName"), equalTo("names"), equalTo("isRelatedTo") )));
+
+    for (Map.Entry<String, Integer> entry : columnIndices.entrySet()) {
```

Comment:
For loops make the test less readable and make it hard to understand why they fail.